src/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import FHIR from 'fhirclient';
+
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
+
+interface App {
+  client?: React.ReactNode;
```

Comment:
The client should be of type `Client`, which is from `import Client from 'fhirclient/lib/Client';`